pkg/util/metrics_helper.go

```diff
@@ -0,0 +1,311 @@
+package util
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// MetricFamiliesPerUser is a collection of metrics gathered via calling Gatherer.Gather() method on different
+// gatherers, one per user.
+// First key = userID, second key = metric name.
+// Value = slice of gathered values with the same metric name.
+type MetricFamiliesPerUser map[string]map[string]*dto.MetricFamily
```

Comment:
Would be great to have basic unit tests on `MetricFamiliesPerUser`

Reply:
I think bucket_store_metrics_test.go already covers all code here. Do you think extra testing is needed here?